platform/linux-generic/odp_timer.c

```diff
@@ -2140,5 +2157,18 @@ int _odp_timer_init_local(void)
 
 int _odp_timer_term_local(void)
 {
+#if CONFIG_TIMER_PROFILE_INLINE
+	if (timer_local.prof_rounds) {
+		int thr = odp_thread_id();
+
+		odp_ticketlock_lock(&timer_global->lock);
+		_ODP_PRINT("Inline timer profiling for thread %i:\n", thr);
+		_ODP_PRINT("  scan rounds:     %" PRIu64 "\n", timer_local.prof_rounds);
+		_ODP_PRINT("  ave scan nsec: %6.1f\n",
```

Comment:
I think no need for the explicit field width, these could be just:
```
_ODP_PRINT("  scan rounds:   %" ...
_ODP_PRINT("  ave scan nsec: %.1f\n" ...
```
so that values are output starting at the same column.